python/src/aiconfig/editor/client/src/components/EditorContainer.tsx

```diff
@@ -579,15 +582,15 @@ export default function EditorContainer({
                 config: event.data,
               });
             }
-          });
-          return;
-        } else {
-          const serverConfigRes = await runPromptCallback(promptName);
+          },
+          enableStreaming
+        );
 
+        if (serverConfigResponse.aiconfig) {
```

Comment:
Does this need to be `serverConfigResponse?.aiconfig`?


Edit: Oh, I guess it's because we have required object with optional aiconfig but maybe it should be `Promise<{aiconfig?: aiconfig} | void>`?